src/containers/SortingView.tsx

```diff
@@ -254,12 +253,21 @@ const mapStateToProps: MapStateToProps<StateProps, OwnProps, RootState> = (state
   hither: state.hitherContext
 })
 
-const mapDispatchToProps: MapDispatchToProps<DispatchProps, OwnProps> = (dispatch: Dispatch<RootAction>, ownProps: OwnProps) => ({
-  onSetSortingInfo: (a: { sortingId: string, sortingInfo: SortingInfo }) => dispatch(setSortingInfo(a)),
-  onSetExternalUnitMetrics: (a: { sortingId: string, externalUnitMetrics: ExternalSortingUnitMetric[] }) => dispatch(setExternalSortingUnitMetrics(a)),
-  onAddUnitLabel: (a: { sortingId: string, unitId: number, label: string }) => dispatch(addUnitLabel(a)),
-  onRemoveUnitLabel: (a: { sortingId: string, unitId: number, label: string }) => dispatch(removeUnitLabel(a)),
-})
+const mapDispatchToProps: MapDispatchToProps<DispatchProps, OwnProps> = (dispatch: Dispatch<RootAction>, ownProps: OwnProps) => {
+  const curationDispatch = (a: SortingCurationAction) => {
+    if (a.type === 'AddLabel') {
```

Comment:
Here is where we translate curation actions for the curation sorting view into global redux actions.